# Parameter msg.

string param_id
float64 param_value
uint8 param_type
uint16 param_count
uint16 param_index
